kb/compiling-and-building/how-do-i-configure-mbedtls.md

```diff
@@ -22,7 +22,7 @@ The `config.py` script automatically finds the `mbedtls_config.h` file when it r
 
 ## Alternative configuration files
 
-You might want to keep the custom configuration file for your application outside the Mbed TLS source tree. You can do this by defining the macro `MBEDTLS_CONFIG_FILE` for the desired filename (including the quote or angular brackets) at compile time. For example, using **make**:
+You might want to keep the custom configuration file for your application outside the Mbed TLS source tree. You can do this by defining the macro `MBEDTLS_CONFIG_FILE` for the desired filename at compile time. For example, using **make**: (Note: The angle brackets `<>` are included.)
```

Comment:
I know what you mean by “The angle brackets `<>` are included” because I know the context, but it doesn't follow from the meaning of the words. Yes, I can see that there are angle brackets in the line below, so what? The point is that _the reader_ needs to include them (or double quotes) in their command.